src/relion/_parser/relativeicethickness.py

```diff
@@ -0,0 +1,98 @@
+import csv
+import logging
+import pathlib
+from collections import namedtuple
+
+from relion._parser.jobtype import JobType
+
+logger = logging.getLogger("relion._parser.relativeicethickness")
+
+RelativeIceThicknessMicrograph = namedtuple(
+    "RelativeIceThicknessMicrograph",
+    ["minimum", "q1", "median", "q3", "maximum", "micrograph_path"],
```

Comment:
```suggestion
    ["micrograph_path", "minimum", "q1", "median", "q3", "maximum"],
```
order has to match that around line 60